_emulator/extensions/export-user-data.env

```diff
@@ -0,0 +1,8 @@
+ALLOWED_EVENT_TYPES=firebase.extensions.export-user-data.v1.firestore
```

Comment:
Would we also require `.env.local` for the emulators to run?

For example if we ran `firebase emulators:start --project=demo-project` from inside the `_emulator` folder.

Reply:
if `.env.local` doesn't exist the emulator will use `.env`